### PR TITLE
Mobile active navigation issues

### PR DESCRIPTION
**Pull request by xameYeoh**

### CODE REVIEW

File: app/src/main/java/pl/llp/aircasting/ui/view/screens/main/MainViewMvcImpl.kt [hunk 1]
```diff
@@ -120,20 +117,27 @@ class MainViewMvcImpl(
...
             }
             true
         }
     }
 
-    private fun goToFollowingTab() {
-        val getFollowedSessionNumber = mSettings?.getFollowedSessionsNumber()
+
+
+    private fun navigateToAppropriateTab() {
```
Comment (LinX64) [inline app/src/main/java/pl/llp/aircasting/ui/view/screens/main/MainViewMvcImpl.kt:130]: Two extra space line above this.

File: app/src/main/java/pl/llp/aircasting/ui/view/screens/main/MainViewMvcImpl.kt [hunk 2]
```diff
@@ -120,20 +117,27 @@ class MainViewMvcImpl(
...
+        val mobileSessionsCount = (mSettings?.mobileActiveSessionsCount() ?: 0)
```
Comment (LinX64) [inline app/src/main/java/pl/llp/aircasting/ui/view/screens/main/MainViewMvcImpl.kt:131]: No need for the parenthesis.

File: app/src/main/java/pl/llp/aircasting/ui/view/screens/main/MainViewMvcImpl.kt (hunk 1 above)
Comment (LinX64) [inline app/src/main/java/pl/llp/aircasting/ui/view/screens/main/MainViewMvcImpl.kt:130]: The best time for using a scope function, how about this?

```
private fun navigateToAppropriateTab() {
        val mobileSessionsCount = mSettings?.mobileActiveSessionsCount() ?: 0
        val followedSessionsCount = mSettings?.getFollowedSessionsCount() ?: 0

        rootActivity.apply {
            if (mobileSessionsCount > 0) {
                goToMobileActiveTab()
                return
            }

            goToFollowingTab()
            adjustMenuVisibility(true, followedSessionsCount)
        }
    }
```

**Review by LinX64 — APPROVED** [review]
I left a small change with a method, overall it looks good.

**Review by LinX64 — APPROVED** [review]

File: app/src/main/java/pl/llp/aircasting/ui/view/screens/main/MainViewMvcImpl.kt (hunk 1 above)
Comment (xameYeoh) [reply app/src/main/java/pl/llp/aircasting/ui/view/screens/main/MainViewMvcImpl.kt:130]: Elegant!

**Pull request merged**